xml/System/ReadOnlySpan`1.xml

```diff
@@ -165,9 +182,13 @@
         <Parameter Name="destination" Type="System.Span&lt;T&gt;" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="destination">The read-only span to copy items into.</param>
+        <summary>Copies the contents of this <see cref="System.ReadOnlySpan`1" /> into a destination <see cref="System.ReadOnlySpan`1" />.</summary>
+        <remarks>
```

Comment:
Are we using the XML comments in the source to seed all of this? e.g.
https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/System/ReadOnlySpan.Fast.cs#L92
There's useful material there and it seems like it's not being considered, but maybe I'm just missing it.